src/test/resources/palaemon-tee-session.yml

```diff
@@ -6,7 +6,7 @@ access_policy:
   read:
    - CREATOR
   update:
-   - CREATOR
+   - NONE
```

Comment:
Should we also add `create_sessions: -NONE` as well, in order not to rely on defaults?

Reply:
By default, it inherits all values defined under `update`. Default value will be `NONE` as well.
It can be explicitly defined, but requires a more thorough rework.
The data model does not support it out of the box and it requires functional dev as well as tests.

Reply:
Right. Let's merge as is!